packages/eas-cli/src/commandUtils/EasCommand.ts

```diff
@@ -175,4 +177,18 @@ export default abstract class EasCommand extends Command {
     await this.analytics.flushAsync();
     return super.finally(err);
   }
+
+  protected baseErrorMessage = 'Command failed.';
+  protected baseGraphQLErrorMessage = 'GraphQL request failed.';
+  protected override catch(err: CommandError): Promise<any> {
+    const isGraphQLError = (err: any): boolean => {
+      return err?.graphQLErrors;
+    };
```

Comment:
It's usually better to place helpers like this outside methods. Also, I'm not sure if we need this particular helper. It's used only once and it's really simple.

Reply:
It is simple, but maybe the logic will change, or we'll need to determine the error type etc. I can remove it too, the same for me

Reply:
I think this is a bit premature for now.